ui/src/components/Publication/Creation/CoAuthor.tsx

```diff
@@ -15,9 +16,25 @@ const CoAuthor: React.FC = (): React.ReactElement => {
 
     const [loading, setLoading] = React.useState(false);
     const [coAuthor, setCoAuthor] = React.useState('');
+    const [emailValidated, setEmailValidated] = React.useState<Boolean>(true);
 
+    const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
(Minor) moving forward it would be good to introduce consistency with return types (here `: void `) when adding new functions/methods.